scripts/exregional_run_prdgen.sh

```diff
@@ -186,7 +186,7 @@ DATA=$COMOUT
 DATAprdgen=$DATA/prdgen_${fhr}
```

Comment:
Shouldn't this file be removed? I'm not sure what it's used for if not RRFS.

Reply:
I think this one and the one below are still needed/useful. This is just about creating subdomains from larger ones. They do happen to be the RRFS domains, but we were suggesting that we'd still support that part of SRW -- domains and physics.